Core/MigrationService.php

```diff
@@ -155,7 +156,7 @@ public function parseMigrationDefinition(MigrationDefinition $migrationDefinitio
      *
      * @todo add support for skipped migrations, partially executed migrations
      */
-    public function executeMigration(MigrationDefinition $migrationDefinition, $useTransaction=true)
+    public function executeMigration(MigrationDefinition $migrationDefinition, $useTransaction = true, $defaultLanguageCode = null)
```

Comment:
I agree, but I couldn't find a better entry point.
For further enhancements, I think `MigrationService` should be refactored.

This worths it for 2.0 IMHO, but you decide 😉